### PR TITLE
Add actions instead of travis config

### PR DESCRIPTION
**Pull request by ssh2go**

### CODE REVIEW

**Review by Calebjh — APPROVED** [review]
Not super familiar with GitHub actions, but this seems okay

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Build
+        run: |
+          make latest && make versioned
+      - name: Push image
+        run: |
+          make push-versioned
```
Comment (drewwells) [inline .github/workflows/push.yml:33]: We should run tests prior to pushing an image. There are ways to kick off actions from other actions, somehow. Easier way may be to combine push in the pr.yml after tests are run.

**Review by drewwells — CHANGES_REQUESTED** [review]
test-gen is downloading `:latest` then running tests on it. We want to verify code that is in the repo in that test. It should build a docker image, then try to build test.proto.

File: .github/workflows/push.yml (hunk 1 above)
Comment (ssh2go) [reply .github/workflows/push.yml:39]: Drew, tests will start in other action (it triggers on each new commit). Please take a look second file in that PR

Comment (Calebjh) [reply .github/workflows/push.yml:39]: The actions run independently though, so even if the tests fail the push action could still work, couldn't it?

Comment (ssh2go) [reply .github/workflows/push.yml:39]: oh, sorry.. that's right. probably I had to make single file with direct dependency (like **needs** keyword)

Comment (Calebjh) [reply .github/workflows/push.yml:39]: I pushed a commit to make it just run the tests for both PR and Push. It seems logical to me that they would be separate actions, but as I noted, I'm not a GitHub Actions expert.

**Review by drewwells — APPROVED** [review]

**Pull request merged**